src/model_filereading.cpp

```diff
@@ -300,17 +312,25 @@ RadiusFileBundle importDataFromRadiusFile(const std::string& radius_path){
 
   std::string line;
   std::ifstream inp_file(radius_path);
+  bool invalid_symbol_detected = false;
   while(getline(inp_file,line)){
     std::vector<std::string> substrings = splitLine(line);
     // substings[0]: Atomic Number
     // substings[1]: Element Symbol
     // substings[2]: Radius
     if(substrings.size() == 3){
-      // TODO: make sure substrings[1] is converted to valid symbol
-      data.rad_map[substrings[1]] = std::stod(substrings[2]);
-      data.atomic_num_map[substrings[1]] = std::stoi(substrings[0]);
+      substrings[1] = strToValidSymbol(substrings[1]);
+      // skip entry if element symbol invalid
+      if (substrings[1].empty()){
+        invalid_symbol_detected = true;
+      }

```

Comment:
If substrings[3] is not a valid number (e.g. start with a letter), the program crashes on windows when loading a structure.

Reply:
I've added a try-catch block to catch the `std::invalid_exception` thrown by `std::stod()`. It would make sense to catch this exception at other places too, where we can't guarantee a valid input.